### PR TITLE
[IMP] sales: add quotation pdf sample

### PR DESCRIPTION
**Pull request by mjvi-odoo**
task-3645924

COM PR: odoo/odoo#152270
ENT PR: odoo/enterprise#61325
UPG PR: odoo/upgrade#5644
DOC PR: odoo/documentation#8391

### CODE REVIEW

File: content/applications/sales/sales/send_quotations/pdf_quote_builder.rst [hunk 1]
```diff
@@ -221,7 +221,10 @@ product PDF that was set to be visible inside the quote, can be viewed and print
 
 .. note::
    Download these :download:`PDF quote builder examples
-   <pdf_quote_builder/pdfquotebuilderexamples.zip>` for added reference.
+   <pdf_quote_builder/pdfquotebuilderexamples.zip>` and
+   :download:`Sample Quotation
+   <pdf_quote_builder/sample_quotation.pdf>`
+   for added reference.
```
Comment (kcv-odoo) [inline content/applications/sales/sales/send_quotations/pdf_quote_builder.rst:227]: IMO sentence should
`Download these` **PDF quote builder examples** `or download` **sample quotation** `for add reference`.

**Review by kcv-odoo — COMMENTED** [review]
@mjvi-odoo Thanks for your work! :1st_place_medal: 
Added a comment :slightly_smiling_face:

**Review by kcv-odoo — APPROVED** [review]
LGTM!

**Comment by Feyensv** [comment]
@robodoo r+

**Pull request closed without merging**